core/templates/dev/head/components/explorationSummaryTile.js

```diff
@@ -42,7 +42,7 @@ oppia.directive('explorationSummaryTile', [function() {
       function($scope, oppiaDatetimeFormatter, ratingComputationService) {
         $scope.getLastUpdatedDatetime = function() {
```

Comment:
Can this function be removed since it's not being used in the exploration summary tile anymore?